// jokes-gco/index.js
var info = [{ 
  id: 1, 
  q: "What do you call a number that can't keep still?",
  a: 'a roamin numeral'
} ,
{ 
  id: 2, 
  q: "What does the zero say to the eight?",
  a: 'Nice belt!'
} ,
{ 
  id: 3, 
  q: "Why are you doing your multiplication on the floor?",
  a: 'You told me not to use tables.'
} ,
{ 
  id: 4, 
  q: "Where did the vampire college student go clothes shopping?",
  a: 'Forever 21'
} ,
{ 
  id: 5, 
  q: "What do snowmen order at fast-food restaurants?",
  a: 'An iceberg-er and fries!'
} ,
{ 
  id: 6, 
  q: " What do snowmen do when they're not feeling well?",
  a: 'They take a chill pill!'
} ,
{ 
  id: 7, 
  q: " How many Dragon Ball Z characters does it take to screw in a lightbulb?",
  a: ' Just one, but it’ll take 6 episodes!'
} ,
{ 
  id: 8, 
  q: " What’s the difference between love and Naruto?",
  a: ' Naruto lasts forever'
} ,
{ 
  id: 9, 
  q: " What is a Pokemon fan’s favorite place in France?",
  a: 'Paras.'
} ,
{ 
  id: 10, 
  q: " What do Saiyans wear to the beach?",
  a: ' Trunks.'
} ,
{ 
  id: 11, 
  q: " How do you make a tissue dance?",
  a: ' You put a little boogie on it.'
} ,
{ 
  id: 12, 
  q: " Knock knock. \n Who's there? \n Cows go. \n Cows go who?",
  a: ' No, cows go moo'
} ,
{ 
  id: 13, 
  q: " did you know the first French fries weren't cooked in France?",
  a: ' they were cooked in Greece'
} ,
{
  id: 14,
  q: "Knock knock. \n Who's there? \n Opportunity.",
  a: "That is impossible. Opportunity doesn’t come knocking twice!"
},
{
  id: 15,
  q: "Why do Java programmers wear glasses?",
  a: "Because they don't C#"
},
{
  id: 16,
  q: "Why did the mushroom get invited to the party?",
  a: "Because he was a fungi."
},
{
  id: 17,
  q: "Why did the mushroom get invited to the party?",
  a: "Because he was a fungi."
},
{
  id: 18,
  q: "I'm reading a book about anti-gravity...",
  a: "It's impossible to put down"
},
{
  id: 19,
  q: "If you're American when you go into the bathroom, and American when you come out, what are you when you're in there?",
  a: "European"
},
{
  id: 20,
  q: "Want to hear a joke about a piece of paper?",
  a: "Never mind...it's tearable"
},
{
  id: 21,
  q: "What do you call a dinosaur that is sleeping?",
  a: "dino-snore!"
},
{
  id:22,
  q:"What is fast, loud and crunchy?",
  a:" rocket chip!"
  
},
{
  id:23,
  q:"Why did the teddy bear say no to dessert?",
  a:"Because she was stuffed."
},
{
id:24,
q:"What has ears but cannot hear?",
a:"cornfield."
},
{
id:25,
q:"What did the left eye say to the right eye?",
a:"Between us, something smells!"
}]
module.exports = {
  get  randomjoke() {
   return info[Math.floor(Math.random() * Math.floor(25))] 
  }
 }
 module.exports.num = function (msg) { 
   if(isNaN(msg)) { 
     console.log("ERROR !!")
  } else {
    var thevalue = msg-1
  return info[thevalue]
  }
};
module.exports.alljoke =  ()=> {  
   return info 
 }
